.github/CODEOWNERS

```diff
@@ -312,7 +312,7 @@ app/models/prescription.rb @department-of-veterans-affairs/vfs-mhv-medications @
 app/models/rate_limited_search.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/models/saml_request_tracker.rb @department-of-veterans-affairs/vsp-identity
 app/models/saved_claim @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/session.rb @department-of-veterans-affairs/vsp-identity
+app/models/saved_claim/veteran_readiness_employment_claim.rb @department-of-veterans-affairs/benefits-non-disability @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
😭 Sorry 1 more comment.
`app/models/session.rb @department-of-veterans-affairs/vsp-identity` got deleted